### ISSUE TITLE
[BUG] run "flutter build ipa" and failed with "module 'better_player' not found"

### ISSUE THREAD

**Issue by windows7lake** [jhomlala/betterplayer#845]
**Describe the bug**
run "flutter build ipa" and failed with "module 'better_player' not found"
It's well when just run in debug mode.

**To Reproduce**
1. run "flutter build ipa":
```
Running Xcode build...                                                  
 └─Compiling, linking and signing...                      1,165ms
Xcode archive done.                                         59.1s
Failed to build iOS app
Error output from Xcode build:
↳
    ** ARCHIVE FAILED **


Xcode's output:
↳
    /Users/yinglin/Workspace/FlutterWorker/meditation/ios/Runner/GeneratedPluginRegistrant.m:12:9: fatal error: module 'better_player' not found
    @import better_player;
     ~~~~~~~^~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in parallel

Encountered error while archiving for device.
```

**Expected behavior**
"flutter build ipa" run well.

**Flutter doctor**
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.8.0, on macOS 11.6 20G165 darwin-x64, locale zh-Hans-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.62.3)
[✓] Connected device (3 available)

• No issues found!

**Better Player version**
- Version: 0.0.79

**Comment by DotCoder21** [comment]
Same Issue , can you help me how to Fix.

**Comment by windows7lake** [comment]
> 

give up and use ijk to play

**Comment by zeshuaro** [comment]
For those who ran into this issue, I was able to resolve it by setting the minimum iOS deployment version to `12.0`

**Comment by PrinzKenny1** [comment]
It's a duplicate of #394 I guess. Solved at least the issue for me. I wasn't running archive in the xcworkspace file but rather in the xcodeproj file.
